docs/content/2.usage/1.nuxt-img.md

```diff
@@ -100,6 +100,8 @@ To generate special versions of images for screens with increased pixel density.
 
 Display a placeholder image before the actual image is fully loaded.
 
+You may also use the [placeholder slot](/usage/nuxt-img#placeholder-1) to pass complex data, if needed.
```

Comment:
question: is this link correct?

Reply:
It should be correct. I've tried it again to make sure it works and it does scroll to the correct position.